### ISSUE TITLE
Cannot write into Excel file

### ISSUE THREAD

**Issue by DanielSanIT** [mganss/ExcelMapper#109]
I am trying to save data from list into exist Excel file with header style.
I do how it showed in #46 , but nothing writes in file.

```cs
public static class Mapper
    {
        public static void Map()
        {
            var products = new List<Product>
               {
                new Product { Name = "Nudossi", NumberInStock = 60, Price = 1.99m },
                new Product { Name = "Halloren", NumberInStock = 33, Price = 2.99m },
                new Product { Name = "Filinchen", NumberInStock = 100, Price = 0.99m },
                };

            var excelMapper = new ExcelMapper("products.xlsx");
            excelMapper.Save("products.xlsx", products, "PROD");

            // This code creates a new file, works fine.
            //new ExcelMapper().Save("products.xlsx", products, "PROD");
        }
    }

    public class Product
    {
        [Column("CustomName")]
        public string Name { get; set; }
        [Column("Custom Number")]
        public int NumberInStock { get; set; }
        [Column("Custom price")]
        public decimal Price { get; set; }
    }
```

**Comment by mganss** [comment]
I can't repro. Can you attach the empty Excel file?

**Comment by DanielSanIT** [comment]
I changed the code to create a new file. It is created a new file with the same style, but still without data.

```cs
var excelMapper = new ExcelMapper("products.xlsx");
excelMapper.Save("products2.xlsx", products, "PROD");
```

[products.xlsx](https://github.com/mganss/ExcelMapper/files/6043177/products.xlsx)
[products2.xlsx](https://github.com/mganss/ExcelMapper/files/6043178/products2.xlsx)

**Comment by DanielSanIT** [comment]
Also, I tried to add "Letter" into model fields. This filled the Excel file with data but without headers.

```cs
    public class Product
    {
        [Column("CustomName", Letter = "A")]
        public string Name { get; set; }
        [Column("Custom Number", Letter = "B")]
        public int NumberInStock { get; set; }
        [Column("Custom price", Letter = "C")]
        public decimal Price { get; set; }
    }
```

![products2](https://user-images.githubusercontent.com/20223013/109160059-5e848c00-7786-11eb-9848-358a9159e4ac.png)

**Comment by mganss** [comment]
If you're mapping by name and saving into a template, the header names must already be there.

**Comment by DanielSanIT** [comment]
Please clarify. Do you mean that if I want to use Excel template, I need to write column headers into the template?

Is there any possibility to write headers into template file through Mapper?

**Comment by mganss** [comment]
I've added a property `CreateMissingHeaders` in 5.1.277 that allows for this use case.

**Comment by DanielSanIT** [comment]
Something went wrong

Model
```cs
public class Product
    {
        [Column("Custom Name", Letter = "C")]
        public string Name { get; set; }
        [Column("Custom NumberInStock", Letter = "D")]
        public int NumberInStock { get; set; }
        [Column("Custom Price", Letter = "E")]
        public decimal Price { get; set; }
    }
```
Without CreateMissingHeaders
```cs
var excelMapper = new ExcelMapper("products.xlsx")
            {
                HeaderRow = false,
                MinRowNumber = 3,
                MaxRowNumber = 5
            };
 excelMapper.Save("products.xlsx", products, "PROD");
```
![NoCreateMissingHeaders](https://user-images.githubusercontent.com/20223013/109198993-f185ec00-77af-11eb-9594-1dabe0552875.png)

With CreateMissingHeaders
```cs
var excelMapper = new ExcelMapper("products.xlsx")
            {
                //HeaderRow = false,
                CreateMissingHeaders = true,
                MinRowNumber = 3,
                MaxRowNumber = 5
            };
excelMapper.Save("products.xlsx", products, "PROD");
```
![CreateMissingHeaders](https://user-images.githubusercontent.com/20223013/109199590-afa97580-77b0-11eb-8460-8a3c81fc6ff6.png)

**Comment by mganss** [comment]
Should be fixed in 5.1.278.

**Comment by DanielSanIT** [comment]
I am not sure that it is right, but I thought that headers should be in row 3
![Capture](https://user-images.githubusercontent.com/20223013/109549276-d0d6d280-7ade-11eb-845d-eb4b03adb620.PNG)
